docs/python/1.6/python-appapi-devguide-5.md

```diff
@@ -67,3 +77,181 @@ def main():
 ~~~~~~
 
 Any transforms that are performed on the parallelized `Stream` object occur in parallel to the degree that is specified in the `.parallel()` function. In this example, you specified 4, which means that four channels process the data in the parallel region on the graph.
+
+
+<a id="consistent_region"></a>
+
+# Consistent region
+
+Because of business requirements, some applications require that all tuples in an application are processed at least once.
+By default any state is reset to its initial state after a processing element (PE) restart. A restart may occur due to:
+
+* a failure in the PE or its resource,
+* a explicit PE restart request,
+* a parallel region width change (IBM Streams 4.3 or later)
```

Comment:
A parallel region width change is not supported by Streams when the parallel region belongs to a consistent region. Therefore, I would NOT mention the third bullet of the list here. For autonomous regions, this bullet is correct.

Reply:
should be removed